src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!--<h1>Search bar airbnb</h1>-->
+    <form action=""
+    class="formBig"
+    method="GET"
+    data-qa="big">
+      <input
+      class="searchBig"
+      type="search"
+      name="searchBig"
+      data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;">
+    </form>
```

Comment:
```suggestion
    <form
      action=""
      class="formBig"
      method="GET"
      data-qa="big">
        <input
          class="searchBig"
          type="search"
          name="searchBig"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
    </form>
```
Let's fix formatting everywhere